enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -208,20 +210,21 @@ func (c *ociContainer) Create(ctx context.Context, workDir string) error {
 	if err := c.createBundle(ctx, pid1); err != nil {
 		return status.UnavailableErrorf("create OCI bundle: %s", err)
 	}
-	// Create container
-	if err := c.invokeRuntimeSimple(ctx, "create", "--bundle="+c.bundlePath(), cid); err != nil {
+	// Create container (discard output as pid 1 is sleep, which inherits and never closes
+	// stdout/stderr)
+	if err := c.invokeRuntimeSimple(ctx, nil, "create", "--bundle="+c.bundlePath(), c.cid); err != nil {
 		return status.UnavailableErrorf("create container: %s", err)
 	}
 	// Start container
-	if err := c.invokeRuntimeSimple(ctx, "start", c.cid); err != nil {
+	if err := c.invokeRuntimeSimple(ctx, &interfaces.Stdio{}, "start", c.cid); err != nil {
 		return status.UnavailableErrorf("start container: %s", err)
 	}
 	return nil
 }
 
 func (c *ociContainer) Exec(ctx context.Context, cmd *repb.Command, stdio *interfaces.Stdio) *interfaces.CommandResult {
 	// TODO: wire up stdio
-	return c.invokeRuntime(ctx, cmd, "exec", "--cwd="+execrootPath, c.cid)
+	return c.invokeRuntime(ctx, cmd, stdio, "exec", "--cwd="+execrootPath, "--detach", c.cid)
```

Comment:
out of curiosity, why do we need `--detach` here? (maybe worth adding a comment for future readers too)